paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -0,0 +1,69 @@
+# INT8 MKLDNN quantization 
+
+This document describes how to use Paddle inference to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKLDNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+
+## 0. Install PaddlePaddle 
+Follow PaddlePaddle [installation instruction](https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification#installation) to install PaddlePaddle. If you build PaddlePaddle yourself, please use the following cmake arguments. 
+```
+cmake ..  -DWITH_TESTING=ON -WITH_FLUID_ONLY=ON -DWITH_GPU=OFF -DWITH_MKL=ON  -WITH_SWIG_PY=OFF -DWITH_INFERENCE_API_TEST=ON -DON_INFER=ON
+
+```  
+Note: MKLDNN and MKL are required.
+
+## 1. Enable INT8 MKLDNN quantization 
+You can refer to the unit test in `analyzer_int8_image_classification_tester.cc`.
```

Comment:
Consider replacing with:
For reference, please examine the code of unit test enclosed in `analyzer_int8_image_classification_tester.cc`.

Reply:
Done